src/linux-dev-certs/CertificateManager.aspnetcore.cs

```diff
@@ -18,19 +20,38 @@ partial class CertificateManager
     private const string ServerAuthenticationEnhancedKeyUsageOid = "1.3.6.1.5.5.7.3.1";
     private const string ServerAuthenticationEnhancedKeyUsageOidFriendlyName = "Server Authentication";
     private const string LocalhostHttpsDnsName = "localhost";
-    private const string LocalhostHttpsDistinguishedName = "CN=" + LocalhostHttpsDnsName;
     public const int RSAMinimumKeySizeInBits = 2048;
     private int AspNetHttpsCertificateVersion => CurrentAspNetCoreCertificateVersion;
-    public string Subject => LocalhostHttpsDistinguishedName;
+    public string Subject => $"OU={Environment.UserName}@{Environment.MachineName},O={AspNetHttpsOidFriendlyName}";
 
     // Copied from aspnetcore CertificateManager.cs.
     // Call to 'CreateSelfSignedCertificate' at the end replaced by 'CreateAspNetDevelopmentCertificate'.
     internal X509Certificate2 CreateAspNetCoreHttpsDevelopmentCertificate(DateTimeOffset notBefore, DateTimeOffset notAfter)
     {
         var subject = new X500DistinguishedName(Subject);
         var extensions = new List<X509Extension>();
+        
         var sanBuilder = new SubjectAlternativeNameBuilder();
         sanBuilder.AddDnsName(LocalhostHttpsDnsName);
+        if (Socket.OSSupportsIPv4)
+        {
+            sanBuilder.AddIpAddress(IPAddress.Loopback);
+        }
+        if (Socket.OSSupportsIPv6)
+        {
+            sanBuilder.AddIpAddress(IPAddress.IPv6Loopback);
+        }
```

Comment:
Why can't the clients refer to the hostname, i.e.: `localhost` instead of having to add ip addresses here?